### DISCUSSION TITLE
Metrics in k3s

### DISCUSSION THREAD

**Discussion by sushiMix** [k3s-io/k3s#8322] [Q&A]
Hello, I'm checking difference between rke2 and k3s behavior.

I have seen that k3s provide a metric endpoint on http://127.0.0.1:10249.
Such port for rke2 provide metrics for kube-proxy (like standard kubernetes).
For k3s, we have also metrics from kubelet which are usually on another secured port.

Is it normal to have provided such metrics from this endpoint ?

**Comment by brandond** [comment]
This comes up occasionally - see the discussion at https://github.com/k3s-io/k3s/issues/3619#issuecomment-1483521268
